invenio_rdm_records/services/pids/providers/base.py

```diff
@@ -152,17 +158,30 @@ def get_status(self, identifier, **kwargs):
         """Get the status of the identifier."""
         return self.get(identifier, **kwargs).status
 
-    def validate(self, identifier=None, provider=None, client=None, **kwargs):
+    def validate(
+        self, record, identifier=None, provider=None, client=None, **kwargs
+    ):
         """Validate the attributes of the identifier.
 
         :returns: A tuple (success, errors). The first specifies if the
                   validation was passed successfully. The second one is an
                   array of error messages.
         """
+        errors = []
         if provider and provider != self.name:
-            errors = [
+            errors.append(
                 _(f"Provider name {provider} does not match {self.name}")
-            ]
-            return False, errors
+            )
+        # deduplication check
```

Comment:
Does it make sense to continue checking here? I think it should simply return here...